src/Components/HistoricalDataChart/HistoricalDataChart.scss

```
@@ -1,36 +1,17 @@
-.chartContainer{
-   display: flex;
-   flex-direction: column;
-   align-items: flex-start;
-   position: absolute;
-   top: 0;
-   right: 0;
-   margin-right: 8%;
-}
-
-.dropdownContainer {
-   margin-right: 200px;
-   align-self: flex-end;
-}
-
-.dateDropdown{
-   padding-left: 8px;
-}
+.chartDateFilter {
+   .dateDropdown{
+      padding-left: 8px;
+      float:right;
+   }
 
-.loadingErrorContainer {
-   width: 756px;
-   height: 275px;
+   :first-child {
+      vertical-align: bottom !important;
+   }
 }
-
-// Media Query
-@media (max-width: 1620px ){
-   .chartContainer {
-      position: static; 
-   } 
-
-    .dropdownContainer {
-      margin-left: 56px;
-      align-self: center;
-      margin-right: 0px;
+.historical-chart-panel {
```

@kgaikwad  I saw the mockup shared by Stefan and I think it will be good to add bit of right margin to the chart to have some space between chart and dropdown. 

```suggestion
.historical-chart-panel {
margin-right: 24px;
```

---

With above suggested change it disturbs the alignment of legendInfo so I will prefer to keep current space to align legend info. Any thoughts @kuklas?

---

im sure i understand, what does it mean that it disturbs the alingment?

---

> what does it mean that it disturbs the alingment?

@kuklas, it disturbs alignment between date dropdown and legendata, as per you shown in above screenshot they are aligned vertically.  With margin-right, legend data will shift towards right direction. If that is fine, I can apply above suggested CSS.

---

@kuklas With current implementation it passes legend attributes to main chart component and no option to set styling specific to legend. 
Referred - https://www.patternfly.org/v4/charts/line-chart/
When I tried to apply CSS to give some space between both side of question mark, legend & date filter will not be vertically aligned. As discussed during UX meeting, we can address any further improvements related to chart separately.